org.hdfgroup.HDFView.yml

```
@@ -0,0 +1,113 @@
+app-id: org.hdfgroup.HDFView
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+build-options:
+  env:
+    JAVA_HOME: /usr/lib/sdk/openjdk/jvm/openjdk-17
+    ANT_HOME: /usr/lib/sdk/openjdk/ant
+finish-args:
+  - --env=PATH=/app/bin:/app/jre/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=x11
```

```suggestion
  - --socket=wayland
  - --socket=fallback-x11
```

assuming it supports wayland.